remote-api/src/main/resources/alfresco/public-rest-context.xml

```diff
@@ -946,6 +946,26 @@
         <property name="ruleSettings" ref="RuleSettings" />
     </bean>
 
+    <bean id="simpleConditionMapper" class="org.alfresco.rest.api.impl.mapper.rules.RestRuleSimpleConditionModelMapper">
+        <constructor-arg name="namespaceService" ref="NamespaceService"/>
+        <constructor-arg name="nodes" ref="Nodes"/>
+    </bean>
+
+
+    <bean id="RestModelMapper" class="org.springframework.aop.framework.ProxyFactoryBean">
```

Comment:
Do we need this bean?  Doesn't the exception handling already happen elsewhere?

Reply:
I thought we needed that one but it seems that not, so removed it.